src/main/java/org/brunocvcunha/instagram4j/requests/payload/InstagramFeedItem.java

```diff
@@ -46,8 +46,8 @@
     public Map<String, Object> attribution;
     public List<ImageMeta> video_versions;
     public ImageVersions image_versions2;
-    public List<InstagramFeedUserTag> usertags;
-    public FeedItemLocation location;
+    public InstagramUserTagsContainer usertags;
+    public Map<String, Object> location;
```

Comment:
Why did you removed FeedItemLocation obj?

Reply:
was moved to InstagramUserTagsContainer. The model structure has changed

Reply:
Thanks but what about FeedItemLocation? Was it necesary to replace "location" variable FeedItemLocation to Map?

Reply:
just realized i misread your first message, sorry. i honestly don't know why i did that. probably had a reason to do so, but i don't know what it was anymore..